tests/foreman/installer/test_installer.py

```diff
@@ -1395,7 +1395,10 @@ def sat_non_default_install(module_sat_ready_rhels):
 @pytest.mark.e2e
 @pytest.mark.tier1
 @pytest.mark.pit_client
-def test_capsule_installation(sat_default_install, cap_ready_rhel, default_org):
+@pytest.mark.parametrize(
+    'setting_update', [f'http_proxy={settings.http_proxy.un_auth_proxy_url}'], indirect=True
+)
+def test_capsule_installation(sat_default_install, cap_ready_rhel, setting_update):
```

Comment:
PRT failed with AssertionError, is this a known issue or due to any BZ? if not, I think it should be reported 
```
        # no errors/failures in /var/log/candlepin/*
        result = satellite.execute(r'grep -iR "error" /var/log/candlepin/*')
>       assert len(result.stdout) == 0
E       assert 2360 == 0
E        +  where 2360 = len('/var/log/candlepin/candlepin.log:2024-02-08 14:39:41,272 [thread=Thread-4 (ActiveMQ-client-global-threads)] [=, org=,...d=] ERROR org.candlepin.audit.DefaultEventMessageReceiver - Unable to process message 97: argument "content" is null\n')
E        +    where '/var/log/candlepin/candlepin.log:2024-02-08 14:39:41,272 [thread=Thread-4 (ActiveMQ-client-global-threads)] [=, org=,...d=] ERROR org.candlepin.audit.DefaultEventMessageReceiver - Unable to process message 97: argument "content" is null\n' = stdout:\n/var/log/candlepin/candlepin.log:2024-02-08 14:39:41,272 [thread=Thread-4 (ActiveMQ-client-global-threads)] [=...udit.DefaultEventMessageReceiver - Unable to process message 97: argument "content" is null\n\nstderr:\n(0, b'')\nstatus: 0.stdout
```

Reply:
It's been reported to the candlepin team and I think they are backporting it. I'll have to check again it's been a little while.